### PR TITLE
Upgrade as -> Prepare project to android studio flamingo

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: data/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.antosik.benchproject.data" />
```
Comment (BorysChajdas) [inline data/src/main/AndroidManifest.xml:2]: Is Manifest for `data` module even needed? 😄 Maybe it could be deleted

File: gradle.properties [hunk 2]
```diff
@@ -20,4 +20,6 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```
Comment (BorysChajdas) [inline gradle.properties]: What is the reason for that change?

File: data/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Nataniel-Antosik) [reply data/src/main/AndroidManifest.xml:2]: I will try to delete it :D

Comment (Nataniel-Antosik) [reply data/src/main/AndroidManifest.xml:2]: I will try to delete it :D

File: gradle.properties (hunk 2 above)
Comment (Nataniel-Antosik) [reply gradle.properties]: I used `AGP Upgrade Assistant` and I don't know why it is changed :D

File: data/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Nataniel-Antosik) [reply data/src/main/AndroidManifest.xml:2]: Done

File: gradle.properties (hunk 2 above)
Comment (Nataniel-Antosik) [reply gradle.properties]: I deleted it 😄

**Review by BorysChajdas — APPROVED** [review]

**Pull request merged**